src/components/LoansByCategory/QuickFilters/QuickFilters.vue

```diff
@@ -181,11 +184,18 @@ export default {
 				kivaUs: false,
 				endingSoon: false,
 			},
+			isMobile: false,
 		};
 	},
 	mixins: [
 		loanChannelQueryMapMixin
 	],
+	mounted() {
+		window.addEventListener('resize', _throttle(() => {
```

Comment:
The `addEventListener` requires an associated `removeEventListener` in `beforeDestroy`. In order for that to work, the method (in this case the return `_throttle`) needs to be a unique instance of the function. So I'm not 100% if our other usages of `_throttle` for checking resize will actually remove the event listener. Please ensure the event listener is removed, or consider using the CSS-only approach mentioned in the other comment.

Reply:
It seems to be working when switching views but yeah I'll double check and copy from the other files I took this from